subsys/bluetooth/controller/include/ll.h

```diff
@@ -206,3 +206,38 @@ void ll_rx_mem_release(void **node_rx);
 void ll_timeslice_ticker_id_get(uint8_t * const instance_index, uint8_t * const user_id);
 void ll_radio_state_abort(void);
 uint32_t ll_radio_state_is_idle(void);
+
+/* Direction Finding */
+
+/* @brief Function sets CTE transmission parameters for a connection.
+ *
+ * @param[in]handle			Connection handle.
+ * @param[in]cte_types			Bitfield holding information about
+ *					allowed CTE types.
+ * @param[in]switching_patterns_len	Numbe of antenna IDs in switching
+ *					pattern.
+ * @param[in]ant_id			Array od antenna IDs that are antenna
+ *					switching pattern.
+ *
+ * @return Status of command completion.
+ */
+uint8_t ll_df_set_conn_cte_tx_params(uint16_t handle, uint8_t cte_types,
+				     uint8_t switching_patterns_len,
+				     uint8_t *ant_id);
+
+/* @brief Function provides information about Direction Finding
+ * antennae switching and sampling related settings.
+ *
+ * @param[out]switch_sample_rates	Pointer to store available antennae
+ *					switch-sampling configurations.
+ * @param[out]num_ant			Pointer to store number of available
+ *					antennae.
+ * @param[out]max_switch_pattern_len	Pointer to store maximum number of
+ *					antennae switch patterns.
+ * @param[out]max_CTE_len		Pointer to store maximum length of CTE
+ *					in [8us] units.
+ */
+void ll_df_read_ant_inf(uint8_t *switch_sample_rates,
+			uint8_t *num_ant,
+			uint8_t *max_switch_pattern_len,
+			uint8_t *max_CTE_len);
```

Comment:
Variable names should be lower-case only.

Reply:
Corrected to lower case